googletest/include/gtest/gtest.h

```diff
@@ -1787,14 +1787,15 @@ template <class R1, class R2, class T>
 AssertionResult FPNearPredFormat(const char* expr1, const char* expr2,
                                  const char* abs_error_expr, const R1& val1,
                                  const R2& val2, const T& abs_error) {
-  using namespace std;
+  using std::fabs;
   auto diff = fabs(val1 - val2);
```

Comment:
lets just spell it put, there is no sense for the using statement 1 line above and there is only 1 usage